README.md

```diff
@@ -128,31 +127,18 @@ The model will only be available internally, but can be accessed externally thro
 > NOTE: The set of instructions in this section are a modified version of the one found on the
 [Image Caption Generator Project Page](https://github.com/IBM/MAX-Image-Caption-Generator)
 
-#### 1. Build the Model
+#### 1. Deploy the Model
 
-Clone the Image Caption Generator model locally. In a terminal, run the following command:
-
-    git clone https://github.com/IBM/MAX-Image-Caption-Generator.git
-
-Change directory into the repository base folder:
-
-    cd MAX-Image-Caption-Generator
-
-To build the docker image locally, run:
+To run the docker image, which automatically starts the model serving API, run:
 
-    docker build -t max-im2txt .
+    docker run -it -p 5000:5000 codait/max-image-caption-generator
 
-All required model assets will be downloaded during the build process.
+This will pull the latest docker image available on Docker Hub to run. If you'd rather build the model locally you can
```

Comment:
Maybe I'm being to nit picky but that command will not always pull the latest version from docker hub. If the user has pulled the image in the past and it is cached locally docker will run that image even if a newer image is available on docker hub.

Reply:
that is correct, maybe I should reword it to clarify that it pulls the latest the first time then reuses that same version from then on. Maybe add `...run, or use a previous copy if you've run the command previously.` what do you think?

Reply:
I really like the simplicity of being able to use the model with a single command and would hate to lose that.

Maybe something like

> This will pull a pre-built image from docker hub (or use an existing image if already cached locally) and run it.

Reply:
ooo, I like that, I'll update to use it

Reply:
updated